src/main/java/pl/simpleascoding/tutoringplatform/service/user/UserServiceImpl.java

```diff
@@ -110,21 +111,53 @@ public String confirmUserRegistration(String tokenValue) {
      * @param username         Username of the user
      * @return The status of the operation
      */
+
     @Override
     @Transactional
-    public String changeUserPassword(ChangeUserPasswordDTO newPasswordsData, String username) {
+    public String changeUserPassword(ChangeUserPasswordDTO newPasswordsData, String username, String rootUrl) {
```

Comment:
Instead `newPasswordsData` should be `changeUserPasswordDTO` or just `dto`.

I know `ChangeUserPasswordDTO ` contain:
- old password, 
- new password, 
- confirmation password
But this object is using for change one password - not to change  many password**s**.